p2p/host.go

```diff
@@ -31,7 +31,7 @@ func DefaultConfig() Config {
 	return Config{
 		Listen:             "/ip4/0.0.0.0/tcp/7513",
 		Flood:              false,
-		MinPeers:           6,
+		MinPeers:           20,
```

Comment:
Why?

Reply:
i want node to connect with more peers than we will have bootnodes (usually we have 10, therefore double), this way they will always eagerly look for other nodes and won't wait until they disconnected from bootnodes

Reply:
Ok understood. thx.